src/components/CreateVmWizard/steps/Networking.js

```diff
@@ -318,14 +344,27 @@ class Networking extends React.Component {
 
   handleCellChange (rowData, field, value) {
     const editingRow = this.state.editing[rowData.id]
+    const editingErrors = {}
     if (editingRow) {
       editingRow[field] = value
-      this.setState(state => ({
-        editing: {
-          ...state.editing,
-          [rowData.id]: editingRow,
-        },
-      }))
+      if (field === 'name') {
+        const { unique, valid } = this.isVnicNameUniqueAndValid(editingRow)
+        const validAndUniqueName = unique && valid
+        editingErrors.nicInvalidName = !valid
+        editingErrors.nicDuplicateName = !unique
+        this.props.onUpdate({ valid: validAndUniqueName }, false)
```

Comment:
Shouldn't need the `false` here if you update the `onUpdate` related functions as suggested in `CreateVmWizard`.